ovirt_imageio/_internal/qemu_nbd.py

```diff
@@ -86,6 +89,15 @@ def url(self):
         return urllib.parse.urlparse(url)
 
     def start(self):
+        # Implement "auto" cache mode, planned for qemu 7.
+        if self.cache is None:
+            self.cache = "none" if self._can_use_direct_io() else "writeback"
+            log.debug("Using cache=%r", self.cache)
```

Comment:
Shouldn't this `log.debug()` be out of the `if` (so it will be printed **always**)?
Same for line #99.

Reply:
No, this is logged only if we cache was automatically set. If the user set the cache to explicit value, we use it as is so there is no need to log.